internal/command/views/show.go

```diff
@@ -89,7 +96,10 @@ func (v *ShowHuman) DisplayState(stateFile *statefile.File, schemas *tofu.Schema
 	return 0
 }
 
-func (v *ShowHuman) DisplayPlan(plan *plans.Plan, planJSON *cloudplan.RemotePlanJSON, config *configs.Config, priorStateFile *statefile.File, schemas *tofu.Schemas) int {
+func (v *ShowHuman) DisplayPlan(ctx context.Context, plan *plans.Plan, planJSON *cloudplan.RemotePlanJSON, config *configs.Config, priorStateFile *statefile.File, schemas *tofu.Schemas) int {
+	_, span := tracing.Tracer().Start(ctx, "Display")
```

Comment:
Same comment as Martin's from this file. Do we really want to record traces here? 🤔